app/views/subscription_mailer/digest.html.erb

```diff
@@ -36,6 +36,19 @@ Dear <%= @user.profile.firstname || @user.username %>,<br/>
   <%= link_to("View all results on #{TeSS::Config.site['title_short']}", subscription_results_url(@subscription)) %>
 </p>
 
+<% if @collections&.any? %>
+<p>
+  Collections you might like to update:
+</p>
+<ul>
+  <% @collections.each do |collection| %>
+    <li>
+      <%= link_to collection.title, send("curate_#{@subscription.subscribable_type.downcase.pluralize}_collection_url", collection) %>
+    </li>
+  <% end -%>
+</ul>
+<% end %>
```

Comment:
Could you add some small test cases to `test/mailers/subscription_mailer_test.rb` that covers this code?

e.g
1. one case where the user is a collaborator on a collection, for an event digest email
2. one case where the user owns a collection for a material digest email

that should cover everything